aiida_quantumespresso/calculations/pw.py

```diff
@@ -62,9 +62,9 @@ def xml_filepaths(cls):
     @classmethod
     def define(cls, spec):
         super(PwCalculation, cls).define(spec)
-        spec.input('metadata.options.input_filename', valid_type=six.string_types, default=cls._DEFAULT_INPUT_FILE, non_db=True)
-        spec.input('metadata.options.output_filename', valid_type=six.string_types, default=cls._DEFAULT_OUTPUT_FILE, non_db=True)
-        spec.input('metadata.options.parser_name', valid_type=six.string_types, default='quantumespresso.pw', non_db=True)
+        spec.input('metadata.options.input_filename', valid_type=six.string_types, default=cls._DEFAULT_INPUT_FILE)
+        spec.input('metadata.options.output_filename', valid_type=six.string_types, default=cls._DEFAULT_OUTPUT_FILE)
+        spec.input('metadata.options.parser_name', valid_type=six.string_types, default='quantumespresso.pw')
```

Comment:
Can you comment on the reason behind this change? And should we apply the same to `NamelistsCalculation`?

Reply:
Because it is superfluous. The `metadata` namespace is already marked as `non_db=True` and this property applies to all ports within it.

Reply:
Ah, good to know.